drivers/hmis/app/models/hmis/hud/project.rb

```diff
@@ -78,4 +79,8 @@ def active
 
     operating_end_date >= Date.today
   end
+
+  def enrollments
+    Hmis::Hud::Enrollment.in_project_including_wip(self)
+  end
```

Comment:
See https://github.com/greenriver/hmis-warehouse/pull/2755/files#r1070190817 for some random thoughts.